README.md

```diff
@@ -1,87 +1,98 @@
 
-The website is split into two components: the API and the front-end. Our immediate focus is developing a solid API that can power multiple platforms. Over the next few weeks, we'll be adding more and more front-end content to complement our API efforts.
+The website, which is of course built in Swift using [Vapor](https://vapor.codes/), is split into two key components:
+1. The frontend which provides the community with information about the event, as well as the capability to buy tickets.
+2. The API which powers our [open-source iOS application](https://github.com/SwiftLeeds/swiftleeds-ios).
+
+## Contributing 🏗
+
+As a community conference, we welcome contributions from anybody: from veteran developers to those just getting started. We're here to help, so feel free to jump into the [#swiftleeds-web](https://swiftleedsworkspace.slack.com/archives/C02M5L9C64D) channel on [our Slack](https://join.slack.com/t/swiftleedsworkspace/shared_invite/zt-wkmr6pif-ZDCdDeHM60jcBUy0BxHdCQ), or raise a [GitHub Issue](https://github.com/SwiftLeeds/swiftleeds-web) if you need a hand.
+
+Equally, feel free to open a new pull request and we can discuss together how to take your ideas forward.
 
 ## Developer Setup 💻
 
-Follow these steps to get your development environment setup to run SwiftLeeds locally. To start, open the `Package.swift` file. It'll take a few moments for the dependencies to download, especially on first open. 
+To get started, follow along with these step by step instructions to get your local development environment setup and ready to run the SwiftLeeds website.
+
+For now, we'll focus on your local environment which is slightly different to how the application is deployed in [production](https://swiftleeds.co.uk/) but allows us to quickly build and test our changes. For advanced users, you can utilise our Dockerfile.
 
 ### Prerequisites
-SwiftLeedsWeb runs off Vapor (of course!), which requires a couple of pre-requisites: 
-* [Homebrew](https://brew.sh)
-* [Vapor](https://docs.vapor.codes/4.0/install/macos/)
-* [Postgres](https://postgresapp.com)
 
-Once you have those two dependencies installed, you're ready to get started. For now, we'll ignore running in a production like environment. Instead, we'll focus on local only build. See the advanced setup below to get started with Docker. 
+SwiftLeeds relies on a few tools before we can get started! 
 
-### Database Setup 
-SwiftLeeds uses Postgres for its database. The following assumes you have postgres installed and ready to use. If not, please follow the tutorial in the link above. 
+* [Xcode](https://developer.apple.com/xcode/) (we recommend using Xcode 14)
```

Comment:
It may be safer to put a specific version of Xcode, or at least Swift, as with multiple developers working on the project, you don't want to be on different versions.